lms/djangoapps/discussion/static/discussion/js/spec/views/discussion_user_profile_view_spec.js

```diff
@@ -1,278 +1,45 @@
-                        page: 4,
-                        numPages: 7,
-                        previous: 3,
-                        first: 1,
-                        leftdots: false,
-                        lowPages: [2, 3],
-                        highPages: [5, 6],
-                        rightdots: false,
-                        last: 7,
-                        next: 5
-                    });
-                });
-
-                it('for middle page of nine (has dots)', function() {
-                    return checkRender({
-                        page: 5,
-                        numPages: 9,
-                        previous: 4,
-                        first: 1,
-                        leftdots: true,
-                        lowPages: [3, 4],
-                        highPages: [6, 7],
-                        rightdots: true,
-                        last: 9,
-                        next: 6
-                    });
-                });
-            });
-
-            describe('pagination interaction', function() {
-                beforeEach(function() {
-                    var deferred;
-                    this.view = makeView(makeThreads(3), 1, 2);
-                    deferred = $.Deferred();
-                    return spyOn($, 'ajax').and.returnValue(deferred);
-                });
-
-                it('causes updated rendering', function() {
-                    $.ajax.and.callFake(function(params) {
-                        params.success({
-                            discussion_data: [
-                                {
-                                    id: 'on_page_42',
-                                    body: 'dummy body'
-                                }
-                            ],
-                            page: 42,
-                            num_pages: 99
-                        });
-                        return {
-                            always: function() {
-                            }
-                        };
-                    });
-                    this.view.$('.discussion-pagination a').first().click();
-                    expect(this.view.$('.current-page').text()).toEqual('42');
-                    return expect(this.view.$('.last-page').text()).toEqual('99');
-                });
-
-                it('handles AJAX errors', function() {
-                    spyOn(DiscussionUtil, 'discussionAlert');
-                    $.ajax.and.callFake(function(params) {
-                        params.error();
-                        return {
-                            always: function() {
-                            }
-                        };
-                    });
-                    this.view.$('.discussion-pagination a').first().click();
-                    return expect(DiscussionUtil.discussionAlert).toHaveBeenCalled();
+            describe('thread list in user profile page', function() {
+                it('should render', function() {
```

Comment:
I understand why we don't need all of the old tests, since we are just using a component that has already been tested. However, it feels like there should be more than just one test here. You could verify that the heading renders correctly (username, counts etc) and you could use a DDT-style test to try it with various headings to ensure that it works with Japanese characters, doesn't suffer from XSS etc.